semgrep-core/src/analyzing/AST_to_IL.ml

```diff
@@ -218,10 +218,75 @@ let composite_of_container : G.container_operator -> IL.composite_kind =
   | Set -> CSet
   | Dict -> CDict
 
+let is_hcl lang =
+  match lang with
+  | Lang.Hcl -> true
+  | _ -> false
+
+let rec hcl_translate_module_call env void (e, args, fields) =
+  (* bTERRAFORM:
+      Terraform modules are translated in the form of `Call`, but we would like to
```

Comment:
Could we add a comment explaining why we didn't do this transformation in Parse_hcl_tree_sitter.ml instead?